helm/ph-ee-engine/values.yaml

```diff
@@ -128,7 +128,7 @@ camunda-platform:
 
 elasticsearch:
   enabled: true
-  imageTag: 7.16.3
+#  imageTag: 7.17.0
```

Comment:
why do we need this change, remove the line as it is not needed

Reply:
https://github.com/elastic/helm-charts/blob/v7.17.3/elasticsearch/values.yaml#L68
It is set as default in the original chart so no need to add it here.